src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.services;
+
+import com.google.common.base.Strings;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.model.out.SampleCase;
+
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+
+@Component
+public class CaseValidator {
+
+    public List<String> getWarnings(SampleCase theCase) {
+        return Strings.isNullOrEmpty(theCase.email)
+            ? singletonList("'email' is empty")
+            : emptyList();
+    }
+}
```

Comment:
Looks like a good candidate for utility object does it not?

Reply:
A proper compenent IMHO.
I inject this one to transformer , and it's quite useful to be able to mock the responses.
Exception record validator is not static either.

Reply:
Why would you want to mock that?